### ISSUE TITLE
Add `precipitation_type`

### ISSUE THREAD

**Issue by jdemaeyer** [jdemaeyer/brightsky#91]
Blocks: #85

- [x] Determine common precipitation types across all observation types
- [x] Add to MOSMIX parser
- [x] Add to SYNOP parser
- [x] Add to current observations parser
- [x] Add to recent precipitation observations parser
- [x] Add tests
- [x] Write migration (including new `enum` for the precipitation type)
- [x] Add to database exports
- [ ] Add to docs sample response

**Comment by jdemaeyer** [comment]
Common types are `none`, `rain`, `sleet`, `snow`.

MOSMIX and Current observations also have codes for `fog`, `hail`, and `thunderstorm` (of which only `hail` is an actual precipitation type...), but those are missing in the `formOfPrecipitation` field in SYNOP and Recent observations.

I'm tempted to split this into two fields `precipitation_type` and `significant_weather`, where the latter will contain `fog`, `hail`, and `thunderstorm` (and be empty for recent/historical weather) and form the base for the `icon` field.
